daprdocs/content/en/reference/api/secrets_api.md

```diff
@@ -34,6 +34,13 @@ Some secret stores have **optional** metadata properties. metadata is populated
 GET http://localhost:<daprPort>/v1.0/secrets/<secret-store-name>/<name>?metadata.version_id=15
 ```
 
+##### Common metadata parameters
+The following optional meta can be used when cache is enabled. And will be ignored if cache is disabled.
+
+Query Parameter | Description
+--------- | -----------
+metadata.refresh_cache | refresh cache data immediately when setting to `"true"`
```

Comment:
```suggestion
metadata.refresh_cache | when this is set to `"true"`, cache data is immediately refreshed.
```

Reply:
@Taction - Given this PR applies to all secret store component types, for discoverability when using a secret store it is best to update **all** the secret store component metadata field. In other words, can you update all the secrets store component listed here https://docs.dapr.io/reference/components-reference/supported-secret-stores/ that this applies to? It is ok to also keep this in the overview section.

Reply:
@msfussell Thanks for your advice. I will update the secrets store components later.

Reply:
Done.